test/test_lang_activation_builder.py

```diff
@@ -0,0 +1,167 @@
+import os
+import shlex
+import subprocess
+import tempfile
+import unittest
+from pathlib import Path
+
+import utils as exaslct_utils  # type: ignore # pylint: disable=import-error
+from exasol_integration_test_docker_environment.testing import utils  # type: ignore
+
+from exasol_script_languages_container_tool.lib.api.get_language_activation_builder import (
+    LanguageDefinitionComponents,
+    get_language_activation_builder,
+)
+
+
+class LanguageActivationBuilderTest(unittest.TestCase):
```

Comment:
I assume the project uses unittest instead of pytest?

Reply:
Yes, it's on our list to convert the tests